build/python_packages.txt

```diff
@@ -0,0 +1,16 @@
+colorlog>=6.7.0
+google>=3.0.0
+lmdb>=1.4.0
+loguru>=0.6.0
+mergedeep>=1.3.4
+pkg_resources>=0.0.0
+protobuf==3.19.0
```

Comment:
Don't need protobuf anymore without Sawtooth.

Reply:
I assume so, (given that this passes the CI tests)

Reply:
i went through most of the imports to pick up this list. will update for post-sawtooth.